.github/ISSUE_TEMPLATE/meeting-deliverables.md

```diff
@@ -7,36 +7,28 @@ title: "Deliverables Review Meeting: <YYYY-MM-DD>"
 
 # 2i2c Deliverables Review meeting
```

Comment:
Let's just call it sprint planning meeting? I want to try align with pre-existing terminology as much as possible.